.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,50 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v5
+        with:
+          cache: false
+          go-version: '1.23'
+      - uses: actions/checkout@v4
+      - name: Run linter with GH action
+        uses: golangci/golangci-lint-action@v6
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: v1.64.8
+
+          # Optional: working directory, useful for monorepos
+          # working-directory: somedir
+
+          # Optional: golangci-lint command line arguments.
+          # args: --issues-exit-code=0
+          args: --config ./golangcilint.yaml --exclude-files ./pkg/verity/verity.go --enable-all ./cmd/... ./pkg/...
```

Comment:
Could you comment on excluding verity.go ?

Reply:
Ah, I see below related to cgo ... can we add that comment to the lint.yaml so we remember why?